src/api.js

```diff
@@ -34,6 +34,22 @@ export function apiCall(state, endpoint, params, options = {}) {
   return fetch(url, options);
 }
 
+export function apiCallWithUrl(state, url, options = {}) {
```

Comment:
Why do we need an extra helper utility for this single purpose, duplicating the API call code here? `fetchMoreSectionComments` seems to be using `api.get`, why don't we use the same here?

Reply:
Actually I should've duplicated the api.get as I see now that the additional boilerplate from the apiCall wasn't necessary to make a successful call.

The idea was that by duplicating some code we would achieve the situation where we can use the "next" -url from the response without first parsing the url and immediately creating it again, which kinda doesn't make sense to me as we end up writing more code and it leaves us more prone to error. Also, we could use the function in the future if we end up doing more pagination somewhere else... But it's fixed now :)